frontend/src/pages/AddEditProject.tsx

```diff
@@ -247,18 +290,11 @@ const StyledTextField = styled(TextField)`
   font-family: "Roboto Light", sans-serif;
 `;
 
-const StyledCheckboxGroup = styled.div`
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-  margin-top: 10px;
+const StyledButton = styled(Button)`
   width: 100%;
+  height: 56px;
```

Comment:
are these css-codes from mui?

Reply:
I think this is normal css code, but I use it to style the "Button", which is a mui component.
If you mean "const StyledButton = styled(Button)" thats a way of styling components in general. If you have questions about that, we can meet in zoom anytime.